ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -48,23 +79,62 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
         <style>
           :host {
             @apply --layout-vertical;
+            transition: all 0.7s;
           }
           paper-checkbox {
-            --paper-checkbox-label-spacing: 10px;
-            margin-bottom: 5px;
+            width: 100%;
+            margin-top: 6px;
+            --paper-checkbox-label-spacing: 8px;
+          }
+          nuxeo-collapsible {
+            --nuxeo-collapsible-heading-padding: 0;
+          }
+          label {
+            @apply --nuxeo-label;
+          }
+          .showButton a {
+            display: block;
+            margin-top: 4px;
           }
         </style>
 
-        <dom-repeat items="{{buckets}}">
+        <dom-if if="[[!collapsible]]">
+          <template>
+            <label>[[label]]</label>
+            <dom-repeat items="{{buckets}}">
+              <template>
+                <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
+                  [[item.label]] ([[item.docCount]])
+                </paper-checkbox>
+              </template>
+            </dom-repeat>
+          </template>
+        </dom-if>
+
+        <dom-if if="[[_isCollapsible(buckets)]]">
           <template>
-            <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
-              [[item.label]] ([[item.docCount]])
-            </paper-checkbox>
+            <nuxeo-collapsible>
+              <label slot="heading">[[label]]</label>
+              <dom-repeat items="{{_visibleBuckets}}">
+                <template>
+                  <paper-checkbox noink checked="{{item.checked}}" on-change="_computeValues">
+                    [[item.label]] ([[item.docCount]])
+                  </paper-checkbox>
+                </template>
+              </dom-repeat>
+              <span hidden$="[[_hideShowMore(buckets)]]" class="showButton">
+                <a href="javascript:void(0);" on-tap="_toggleShow">
+                  [[_computeShowMoreLabel(_showAll, i18n)]]
+                </a>
+              </span>
+            </nuxeo-collapsible>
           </template>
-        </dom-repeat>
+        </dom-if>
+
         <dom-if if="[[_isEmpty]]">
           <template>
-            <label>[[i18n('checkboxAggregation.noResults')]]</label>
+            <label>[[label]]</label>
+            <span>[[i18n('checkboxAggregation.noResults')]]</span>
```

Comment:
If I'm seeing it correctly, the "non collapsible and empty" scenario will end up with two `<label>[[label]]</label>`.
Guess we can improve the DOM here.